python/CMakeLists.txt

```diff
@@ -36,6 +36,16 @@ declare_mlir_dialect_python_bindings(
   DIALECT_NAME AIE
 )
 
+declare_mlir_dialect_python_bindings(
+  ADD_TO_PARENT AiePythonSources.Dialects
+  ROOT_DIR "${AIE_PYTHON_ROOT_DIR}"
+  TD_FILE dialects/AiexBinding.td
```

Comment:
Generally, I'd prefer "AIEX" to "Aiex"  (but I'd also prefer 'AIE' to 'Aie')

Reply:
Agreed. I've done some renaming.